tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -283,9 +283,6 @@
             os_net_config_iface: {{ dataplane_os_net_config_iface | default ('nic1') }}
             os_net_config_set_route: {{ dataplane_os_net_config_set_route | default(true) | bool }}
             os_net_config_dns: {{ dataplane_os_net_config_dns | default("") }}
-            service_net_map:
```

Comment:
similar to [architecture](https://github.com/openstack-k8s-operators/architecture/pull/250) not cleaning debug and selinux flags?

Reply:
Yeah, you're right. I added those extra variables to the ones I'm removing.